components/board/pi5/board.go

```diff
@@ -0,0 +1,26 @@
+// Package pi5 implements a raspberry pi 5 board.
+package pi5
+
+import (
+	"github.com/pkg/errors"
+	"periph.io/x/host/v3"
+
+	"go.viam.com/rdk/components/board/genericlinux"
+	"go.viam.com/rdk/logging"
+)
+
+const modelName = "pi5"
```

Comment:
Given how many non-raspberry "pi" boards there are, I wonder if this name should be changed. I went with "pi5" for consistency with the pre-existing "pi" component, but it's ambiguous whether that's a raspberry/orange/rock/banana/etc. pi.

Reply:
We can alias it later. Also document it in docs.